srcpkgs/booster/template

```diff
@@ -0,0 +1,49 @@
+# Template file for 'booster'
+pkgname=booster
+version=0.9
+revision=1
+build_style=go
+go_import_path=github.com/anatol/booster
+go_package="${go_import_path}/generator ${go_import_path}/init"
+hostmakedepends="git ruby-ronn"
+depends="busybox-static"
+short_desc="Fast and secure initramfs generator"
+maintainer="travankor <travankor@tuta.io>"
+license="MIT"
+homepage="https://github.com/anatol/booster"
+distfiles="https://github.com/anatol/booster/archive/${version}.tar.gz"
+checksum=786b89ac5f5b6fb063ff58b490525c9fda9c6f2ae53ab06416b25993bdb72ca4
+conf_files="/etc/booster.yaml"
+alternatives="
+ initramfs:/etc/kernel.d/post-install/20-initramfs:/usr/libexec/booster/kernel-hook-postinst
+ initramfs:/etc/kernel.d/post-remove/20-initramfs:/usr/libexec/booster/kernel-hook-postrm
+"
+
+do_build() {
+	cd generator
+	go install -p "$XBPS_MAKEJOBS" -mod=readonly -x -tags "${go_build_tags}" -ldflags "${go_ldflags}"
+
+	cd ../init
+	CGO_ENABLED=0 go install -p "$XBPS_MAKEJOBS" -mod=readonly -x -tags "${go_build_tags}" -ldflags "${go_ldflags}"
+}
+
+do_install() {
+	if [ "$CROSS_BUILD" ]; then
+		vbin ${GOPATH}/bin/linux_${GOARCH}/generator booster
+		vinstall ${GOPATH}/bin/linux_${GOARCH}/init 755 usr/lib/booster
+	else
+		vbin ${GOPATH}/bin/generator booster
+		vinstall ${GOPATH}/bin/init 755 usr/lib/booster
+	fi
+	ronn docs/manpage.md
+	vman docs/manpage.1 booster.1
+	echo "busybox: true" > booster.yaml
+}
+
+post_install() {
```

Comment:
There's not much utility in a separate `post_install` when you are already defining `do_install`, so you might as well do all of this in one place.